payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -303,6 +303,13 @@ constructor(
             isVoucher = false,
             requiresMandate = false,
             hasDelayedSettlement = false
+        ),
+        Affirm(
```

Comment:
@maxwellclemens-stripe might have the same change here. One of us might need to remove this.

Reply:
Yep, I've got the same thing here - first to merge gets to keep it! 🙃